scripts/catalog-tests/generate-tdr-snapshot-metadata.py

```diff
@@ -0,0 +1,85 @@
+#------------------------------------------------------------------------------
+# Generate TDR Snapshot Metadata
+#
+# This script adds snapshots to the catalog as the test user from DC-218,
+# pulling the metadata from the resources JSON file
+# 
+# Before Running:
+#   python3 -m pip install requests argparse
+#   
+#   Authorization: Access the password in the vault using the appropriate command:
+#   vault read secret/dsde/terra/kernel/dev/dev/catalog/tests/user
```

Comment:
You might need to run `vault login -method=github token=$(cat ~/.github-token)` or equivalent beforehand to authenticate.

Reply:
Added!